packages/pigeon/lib/dart_generator.dart

```diff
@@ -133,26 +147,29 @@ void _writeFlutterApi(Indent indent, Api api,
 
 /// Generates Dart source code for the given AST represented by [root],
 /// outputting the code to [sink].
-void generateDart(Root root, StringSink sink) {
+void generateDart(DartOptions opt, Root root, StringSink sink) {
   final List<String> customClassNames =
       root.classes.map((Class x) => x.name).toList();
   final Indent indent = Indent(sink);
   indent.writeln('// $generatedCodeWarning');
   indent.writeln('// $seeAlsoWarning');
   indent.writeln(
       '// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import');
-  indent.writeln('// @dart = 2.8');
+  indent.writeln('// @dart = ${opt.isNullSafe ? '2.10' : '2.8'}');
   indent.writeln('import \'dart:async\';');
   indent.writeln('import \'package:flutter/services.dart\';');
   indent.writeln(
       'import \'dart:typed_data\' show Uint8List, Int32List, Int64List, Float64List;');
   indent.writeln('');
 
+  final String nullBash = opt.isNullSafe ? '!' : '';
```

Comment:
You mean nullBang perhaps? Or maybe 'bash' is another colloquialism I'm missing. `nullExclamation` could be clearer maybe?

Reply:
You're right, it was a mistake, fixed it.